tests/components/rfxtrx/test_init.py

```diff
@@ -104,3 +106,19 @@ def record_event(event):
             "values": {"Sound": 9, "Battery numeric": 0, "Rssi numeric": 7},
         },
     ]
+
+
+async def test_send(hass, rfxtrx):
+    """Test configuration."""
+    assert await async_setup_component(
+        hass, "rfxtrx", {"rfxtrx": {"device": "/dev/null"}}
+    )
+    await hass.async_block_till_done()
+
+    await hass.services.async_call(
+        "rfxtrx", "send", {"event": "0a520802060101ff0f0269"}, blocking=True
+    )
+
+    assert rfxtrx.transport.send.mock_calls == [
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5844602)